examples/radio/radio-rating.html

```diff
@@ -0,0 +1,290 @@
+                  <use class="star" xlink:href="#star" x="50" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="48" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="3" aria-label="three stars" class="star-3 star-4 star-5">
+                  <use class="star" xlink:href="#star" x="90" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="88" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="4" aria-label="four stars" class="star-4 star-5">
+                  <use class="star" xlink:href="#star" x="130" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="128" y="5"/>
+                </g>
+
+                <g role="radio" aria-checked="false" tabindex="-1" data-rating="5" aria-label="five stars" class="star-5">
+                  <use class="star" xlink:href="#star" x="170" y="7"/>
+                  <use class="focus-ring" xlink:href="#focus-ring" x="168" y="5"/>
+                </g>
+              </svg>
+            </div>
+          </div>
+          <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+    </section>
+
+    <section>
+      <h2>Accessibility Features</h2>
+      <ul>
+        <li>
+          To highlight the interactive nature of the rating stars a focus ring appears around the star icon that has focus.  When a pointer hovers over a star the cursor changes to a pointer and the number of filled stars is updated to include the hovered star.  When a pointer leaves a star, the filled stars are restored to indicate the currently checked star.
+        </li>
+        <li>
+          To ensure the borders of the stars and the focus ring have sufficient contrast with the background when high contrast settings invert colors, the color of the borders are synchronized with the color of the text content.
+          For example, the color of the focus ring border is set to match the foreground color of  high contrast mode text by specifying the CSS <code>currentColor</code> value for the <code>stroke</code> property of the inline SVG <code>polygon</code> to draw the star borders and <code>rect</code> element used to draw the focus ring border.
+          To make the background of the <code>polygon</code> and <code>rect</code> elements match the high contrast background color, the <code>fill-opacity</code> attribute of the <code>rect</code> is set to zero.
+          If specific colors were instead used to specify the <code>stroke</code> and <code>fill</code> properties, those colors would remain the same in high contrast mode, which could lead to insufficient contrast between the rail and the thumb or even make them invisible if their color matched the high contrast mode background.<br>
```

Comment:
Copy/paste issue here. There is no thumb or rail. Good to double check this entire paragraph to make sure the appropriate elements are being described.